mlflow/pytorch/__init__.py

```diff
@@ -530,6 +635,7 @@ class LinearNNModel(torch.nn.Module):
         model_data=model_data_subpath,
         pytorch_version=torch.__version__,
         **torchserve_artifacts_config,
+        state_dict=False,
```

Comment:
Can we remove this field?

Reply:
Yes. its no longer needed. will remove it right away